### PR TITLE
Update glossary.yml

### PR DESCRIPTION
**Pull request by feddelegrand7**
## Author: 

- Mohamed El Fodil Ihaddaden

## Language: 

- French

## Terms defined:

- Anonymous function
- Anchor
- 68-95-99.7 rule
- Absolute error
- Abondonware

## Editor
@fmichonneau

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -28,6 +36,10 @@
     term: "afgooiware"
     def: >
       Sagteware wat nie meer onderhou word nie.
+  fr:
+    term: "abandonware"
+    def: >
+      Un logiciel qui n'est plus maintenue.
```
Comment (timtomch) [inline glossary.yml:42]: Small grammatical error. Should be "Un logiciel qui n'est plus maintenu." (without trailing 'e')

**Review by timtomch — CHANGES_REQUESTED** [review]
Looking good. I flagged a small grammar typo on "abandonware".

File: glossary.yml (hunk 1 above)
Comment (feddelegrand7) [reply glossary.yml:42]: Ah yes ! sorry.

**Review by fmichonneau — COMMENTED** [review]
a few suggested edits for consistency.

**Comment by feddelegrand7** [comment]
Thanks :)

**Review by fmichonneau — APPROVED** [review]

**Comment by fmichonneau** [comment]
Thank you!

**Pull request merged**